trio/_socket.py

```diff
@@ -481,7 +584,7 @@ def __init__(self, sock: _stdlib_socket.socket):
         "share",
     }
 
-    def __getattr__(self, name):
+    def __getattr__(self, name: str) -> Any:
```

Comment:
Should probably hide this behind `TYPE_CHECKING`,  then add stubs for each of those methods/properties.

Reply:
ooor just get rid of `forward` altogether.